### ISSUE TITLE
 Question: Is there a spec to define the comment? 

### ISSUE THREAD

**Issue by yanganto** [yasammez/nachricht#2]
Hi, here

This is awesome to make the data readable and also typed in a small size.

It is nice to providing comment features in a readable data format.

Is there a plan to define the comment symbols, such as `//`.

Thanks.

**Comment by yasammez** [comment]
I suppose you mean within the textual representation? That can be done; however, it wouldn't make much sense to include comments in the wire format and thus they would get lost after a roundtrip.

**Comment by yanganto** [comment]
Yup,  There may be one reason for allowing to writing a comment that some people using `yaml` instead of `json`.  If you take this into consideration will be nice, and this just a suggestion.  :)
